src/views/bootablevolumes/list/components/BootableVolumesRow.tsx

```diff
@@ -1,49 +1,67 @@
 import React, { FC } from 'react';
 
+import { isBootableVolumePVCKind } from '@catalog/CreateFromInstanceTypes/utils/utils';
+import { PersistentVolumeClaimModel } from '@kubevirt-ui/kubevirt-api/console';
+import DataSourceModel, {
+  DataSourceModelGroupVersionKind,
+} from '@kubevirt-ui/kubevirt-api/console/models/DataSourceModel';
 import { V1beta1DataSource } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
 import { V1alpha2VirtualMachineClusterPreference } from '@kubevirt-ui/kubevirt-api/kubevirt';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { ANNOTATIONS } from '@kubevirt-utils/resources/template';
 import { isDataSourceCloning } from '@kubevirt-utils/resources/template/hooks/useVmTemplateSource/utils';
 import { NO_DATA_DASH } from '@kubevirt-utils/resources/vm/utils/constants';
 import {
-  K8sGroupVersionKind,
+  getGroupVersionKindForModel,
   ResourceLink,
   RowProps,
   TableData,
-  Timestamp,
 } from '@openshift-console/dynamic-plugin-sdk';
 import { Label, Split, SplitItem } from '@patternfly/react-core';
 import { TableText, WrapModifier } from '@patternfly/react-table';
 
 import BootableVolumesActions from '../../actions/BootableVolumesActions';
-import { getDataSourcePreferenceLabelValue, getPreferenceReadableOS } from '../../utils/utils';
+import { BootableResource } from '../../utils/types';
+import { getPreferenceReadableOS, getSourcePreferenceLabelValue } from '../../utils/utils';
+
+const PersistentVolumeClaimModelGroupVersionKind = getGroupVersionKindForModel(
+  PersistentVolumeClaimModel,
+);
 
 const BootableVolumesRow: FC<
   RowProps<
-    V1beta1DataSource,
+    BootableResource,
     {
-      groupVersionKind: K8sGroupVersionKind;
       preferences: V1alpha2VirtualMachineClusterPreference[];
     }
   >
-> = ({ obj, activeColumnIDs, rowData: { groupVersionKind, preferences } }) => {
+> = ({ obj, activeColumnIDs, rowData: { preferences } }) => {
   const { t } = useKubevirtTranslation();
 
   return (
     <>
-      <TableData id="name" activeColumnIDs={activeColumnIDs} className="pf-m-width-15">
+      <TableData id="name" activeColumnIDs={activeColumnIDs} className="pf-m-width-20">
         <Split hasGutter>
           <SplitItem>
-            <ResourceLink groupVersionKind={groupVersionKind} name={obj?.metadata?.name} />
+            <ResourceLink
+              groupVersionKind={
+                isBootableVolumePVCKind(obj)
```

Comment:
I just remembered you can use this:
https://github.com/kubevirt-ui/kubevirt-plugin/blob/318c9c2180e53edea29a14e9d86b13b7429981a4/src/views/catalog/CreateFromInstanceTypes/utils/utils.ts#L144-L147

Reply:
Great :-)